src/CoreServer.cpp

```diff
@@ -448,7 +448,9 @@ void CoreServer::_setup_routes(const PrometheusConfig &prom_config)
                 try {
                     auto [policy, lock] = _registry->policy_manager()->module_get_locked(p_mname);
                     auto scope = resource.add_scope_metrics();
-                    scope->mutable_scope()->set_name(p_mname);
+                    auto attr = scope->mutable_scope()->add_attributes();
```

Comment:
nice!
with this the scope name should have something like "pktvisorreceiver"
right @weyrick ?

Reply:
@leoparente can we make this "pktvisor/POLICYNAME" please?

Reply:
sure

Reply:
done